assets/responsibleai/environments/responsibleai-vision-py38/context/conda_dependencies.yaml

```diff
@@ -7,7 +7,7 @@ channels:
 dependencies:
   - python=3.8
   - pip
-  - pytorch
+  - pytorch>=1.13.1
```

Comment:
Shouldn't this be installing latest version already anyway since it's not pinned?  I wonder if automl is downgrading it later.  Do we know when they will upgrade and to what version?